copier/config/user_data.py

```diff
@@ -108,8 +112,38 @@ def cast_answer_type(answer: Any, type_fn: Callable) -> Any:
         return answer
 
 
+def render_value(value: Any, env: SandboxedEnvironment, context: AnyByStrDict) -> str:
+    """Render a single templated value using Jinja."""
```

Comment:
Let's make this even more obvious:

```suggestion
    """Render a single templated value using Jinja.

    If the value cannot be used as a template, it will be returned as is.
    """
```